pkg/query/symbol.go

```diff
@@ -56,6 +31,43 @@ func SiblingSymbols(doc DocumentContent, node, before *sitter.Node) []Symbol {
 	return symbols
 }
 
+func ExtractAssignment(doc DocumentContent, n *sitter.Node) Symbol {
```

Comment:
```suggestion
func ExtractAssignee(doc DocumentContent, n *sitter.Node) Symbol {
```

or `ExtractAssignmentLHS`, or some name to make it easier to figure out what is being returned without reading the code.

Reply:
`ExtractVariableAssignment`?